installers/ansible/roles/pgo-operator/tasks/main.yml

```diff
@@ -187,32 +207,46 @@
         - create_rbac | bool
         - pgo_image_pull_secret_manifest != ''
 
-    - name: Create cluster-admin Cluster Role Binding for PGO Service Account
-      command: |
-        {{ kubectl_or_oc }} create clusterrolebinding pgo-cluster-admin \
-          --clusterrole cluster-admin \
-          --user system:serviceaccount:{{ pgo_operator_namespace }}:postgres-operator
-      when: create_rbac|bool and pgo_cluster_admin|bool
+    - name: ClusterRolebinding for PGO Service Account
       tags:
         - install
         - update
+        - test
+      when: create_rbac|bool and pgo_cluster_admin|bool
+      block:
+        - name: Check cluster-admin Cluster Role Binding for PGO Service Account
+          shell: "{{ kubectl_or_oc }} get clusterrolebinding pgo-cluster-admin"
+          register: pgo_cluster_admin_result
+          failed_when: false
+
+        - name: Create cluster-admin Cluster Role Binding for PGO Service Account
+          command: |
+            {{ kubectl_or_oc }} create clusterrolebinding pgo-cluster-admin \
+              --clusterrole cluster-admin \
+              --user system:serviceaccount:{{ pgo_operator_namespace }}:postgres-operator
```

Comment:
This might need quotes when the operator namespace is strange:

```
--user "system:serviceaccount:{{ pgo_operator_namespace }}:postgres-operator"
```

Better might be to use the flag for service accounts:

```
--serviceaccount "{{ pgo_operator_namespace }}:postgres-operator"
```